griptape/utils/observability/griptape_cloud_observability.py

```diff
@@ -0,0 +1,59 @@
+import atexit
+import os
+
+from griptape.utils.observability.griptape_cloud_span_exporter import GriptapeCloudSpanExporter
+from griptape.utils.observability.griptape_instrumentor import GriptapeInstrumentor
+from opentelemetry import trace
+from opentelemetry.trace import Status, StatusCode
+from opentelemetry.instrumentation.threading import ThreadingInstrumentor
+from opentelemetry.sdk.resources import Resource
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import BatchSpanProcessor
+
+
+class GriptapeCloudObservability:
+    def __init__(self):
+        self.trace_provider = None
+        self.root_span_context_manager = None
+        self.exit_handler = None
```

Comment:
Is there a reason this isn't an attrs class?

Reply:
Changed abstractions, which also now use attrs.